kDriveCore/Data/Cache/DriveInfosManager.swift

```diff
@@ -102,7 +102,9 @@ public class DriveInfosManager {
 
     public func getRealm() -> Realm {
```

Comment:
can we use `RealmAccessible` instead (it would do the refresh corectly also)?

Reply:
Yes but in an other PR as this protocol is only available in MailCore